Libs/PandoraSharp/Song.cs

```diff
@@ -49,7 +49,6 @@ public Song(Pandora p, JToken song)
 
             AmazonAlbumID = (string)song["amazonAlbumDigitalAsin"];
             AmazonTrackID = (string)song["amazonSongDigitalAsin"];
-            AmazonAlbumUrl = (string)song["amazonAlbumUrl"];

```

Comment:
These changes in this file will have to be reverted, as they fixed an issue in another pull request.